clippy_lints/src/utils/conf.rs

```diff
@@ -107,7 +107,7 @@ macro_rules! define_Conf {
 pub use self::helpers::Conf;
 define_Conf! {
```

Comment:
I would explain in the documentation why "bar" isn't in that list, as it's a well-known list and I wouldn't be surprised if someone raised the issue or opened a PR to readd it "because we forgot about it".

Reply:
Good idea, I'll try reinforcing the situation in the comments. Thanks!